packages/components/src/components/Avatar/index.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import css from '@styled-system/css';
+import { User } from '@codesandbox/common/lib/types';
+import styled from 'styled-components';
+import { Element } from '../Element';
+import { Text } from '../Text';
+
+interface IAvatarProps {
+  user: User;
+}
+
+export const AvatarImage = styled.img(
+  css({
+    width: 8,
+    height: 8,
+    borderRadius: 'small',
+    border: '1px solid rgba(255, 255, 255, 0.46)',
```

Comment:
explain your color choice please

Reply:
it was color choice on design, we can hate it togheter ahah

Reply:
We'll hate with danny as well, it might or might not be intentional

Reply:
Ill make it the `sidebar.border`

Reply:
Changed in design by Danny